ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/only_local_vlan_trunk_groups.schema.yml

```diff
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  only_local_vlan_trunk_groups:
+    documentation_options:
+      filename: Fabric Variables
+      table: VLAN Settings
+    type: bool
+    description: |
+      A vlan can have many trunk_groups assigned. To avoid unneeded configuration changes on all leaf
+      switches when a new trunk group is added, this feature will only configure the vlan trunk groups
+      matched with local connected_endpoints.
+      See "Details on only_local_vlan_trunk_groups" below.
```

Comment:
add a comment to revisit doc later

Reply:
All descriptions will be reviewed in the next rounds of PRs where we focus on building the documentation.